### ISSUE TITLE
Fine Tuning on our custom dataset

### ISSUE THREAD

**Issue by Arnab1181412** [MengyangPu/EDTER#91]
Hi I have image pairs for my own data and I want to fine tune using this dataset. There are around 500 images of 255 X 255 size. How do I do it?
Thanks in advance

**Comment by MengyangPu** [comment]
@Arnab1181412 
The process is the same as [the training process](https://github.com/MengyangPu/EDTER?tab=readme-ov-file#2-training):
1 Data augment, and generate a [train_pair.txt](https://github.com/MengyangPu/EDTER/blob/main/data/BSDS/ImageSets/train_pair.txt) of your data.
2 Modify the path in the [configuration file](https://github.com/MengyangPu/EDTER/blob/main/configs/bsds/EDTER_BIMLA_320x320_80k_bsds_bs_8.py).
3 Adjust parameters, e.g., learning rate, and batch size.

**Comment by Arnab1181412** [comment]
Thank you for your help. Just had one question: should I change the
path at this
line
<https://github.com/MengyangPu/EDTER/blob/3fe76f3d938206ef9dc8b857a9767b8cd3d28fc7/configs/bsds/EDTER_BIMLA_320x320_80k_bsds_bs_8.py#L3C66-L3C66>.
?

Regards

**Comment by MengyangPu** [comment]
Yes, and you could create your dataset.py based on [configs/_base_/datasets/bsds.py](https://github.com/MengyangPu/EDTER/blob/3fe76f3d938206ef9dc8b857a9767b8cd3d28fc7/configs/_base_/datasets/bsds.py).

**Comment by Arnab1181412** [comment]
Thank you for your help. Let me try this. I will get back to you if I have
any questions.

Regards

**Comment by Arnab1181412** [comment]
Hi I tried following the steps you told me but it didn't work. I did it on
a kaggle notebook and the linux commands are not working. That is the first
training process 2.1.
